packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -59,10 +67,6 @@ interface AutocompleteProps
   readonly placeholder: string;
 }
 
-/**
- * Max statements disabled here to make room for the
```

Comment:
Are we okay removing this comment? It feels like we should probably add context as to why we're disabling lint checks in our commit messages.

Reply:
I think the current standard right now (Jobber wise) is to increase the limit on this 1 function instead of disabling it and then adding a quick summary as to why.

Reply:
Sure, I'll add the comment back in! Going to amend the commit.

Reply:
Pushed an amendment [here](https://github.com/GetJobber/atlantis/compare/8cfc2ba5a39e01b1a81cc142ccbe38b5271332c3..c72d41bb76d278b663577a40b6f5714589eb47be)

Reply:
muchas gracias mi amigo!